### PR TITLE
Backport CASSANDRA-7950 (Support long names in nodetool output)

### PR DESCRIPTION
**Pull request by smiklosovic**
patch by Kurt Greaves; reviewed by Stefan Miklosovic for CASSANDRA-14162

### CODE REVIEW

File: src/java/org/apache/cassandra/tools/nodetool/CompactionHistory.java [hunk 1]
```diff
@@ -41,19 +46,79 @@ public void execute(NodeProbe probe)
...
+        {
+            table.add(eachChc.getAllAsArray());
+        }
+        table.printTo(out);
+    }
+
+    /**
+     * Allows the Compaction History output to be ordered by 'compactedAt' - that is the
+     * time at which compaction finished.
+     */
```
Comment (ekaterinadimitrova2) [inline src/java/org/apache/cassandra/tools/nodetool/CompactionHistory.java:82]: Not sure about this new order that we should introduce it here, I think this goes beyond the scope of what we wanted about the formatting

Comment (smiklosovic) [reply src/java/org/apache/cassandra/tools/nodetool/CompactionHistory.java:82]: @ekaterinadimitrova2  It is trully just backport of the other stuff. Check other branches. I havent changed anything (nor the guy who backported it first)

File: src/java/org/apache/cassandra/tools/nodetool/CompactionHistory.java [hunk 2]
```diff
@@ -41,19 +46,79 @@ public void execute(NodeProbe probe)
         TabularData tabularData = probe.getCompactionHistory();
         if (tabularData.isEmpty())
         {
-            out.printf("There is no compaction history");
+            out.println("There is no compaction history");
```
Comment (ekaterinadimitrova2) [inline src/java/org/apache/cassandra/tools/nodetool/CompactionHistory.java]: I would leave it printf to be consistent as everywhere else. Also, println adds line separator string

**Review by ekaterinadimitrova2 — APPROVED** [review]
Based on the discussions, +1 on green CI.
Feel free to address my only comment on commit, thanks!

**Pull request closed without merging**